earth2studio/lexicon/wb2.py

```diff
@@ -141,8 +141,20 @@ def get_item(cls, val: str) -> tuple[str, Callable]:
         """Return name in WeatherBench vocabulary."""
         wb2_key = cls.VOCAB[val]
 
-        def mod(x: np.array) -> np.array:
-            """Modify name (if necessary)."""
-            return x
+        if wb2_key.split("::")[0] == "relative_humidity":
+
+            def mod(x: np.array) -> np.array:
+                """Relative humidty in WeatherBench uses older calculation and does
+                not scale by 100 natively. Not recommended for use, see IFS method for
+                more modern calculation.
+                https://github.com/google-research/weatherbench2/blob/main/weatherbench2/derived_variables.py#L468
+                """
```

Comment:
Should this return a warning to the user in addition to the docstring? Who is going to read this?

Reply:
I added a little warning into the docs instead of the code.

Dont want too much responsibility of making sure stuff is the same.